src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Tags and properties inside tags should be in alphabetical order throughout the file.

Reply:
Done